frontend/app/(tabs)/_layout.tsx

```diff
@@ -1,33 +1,53 @@
 import { Tabs } from "expo-router";
 import { View } from "react-native";
+import { Ionicons } from '@expo/vector-icons'; // Import Ionicons from @expo/vector-icons
+
 
 function _layout() {
   return (
     // If login is successful, show the Tabs
     <Tabs>
       <Tabs.Screen
         name="feed"
-        options={{ tabBarLabel: "Feed", headerTitle: "Feed" }}
+        options={{ 
+          tabBarLabel: "Feed", 
+          headerTitle: "Feed",
+          tabBarIcon: ({ color, size }) => (
+            <Ionicons name="home" size={30} color={color} /> // Change the icon here
+          ),
+        }}
       />
       <Tabs.Screen
         name="search"
-        options={{ tabBarLabel: "Search", headerTitle: "Search" }}
+        options={{ tabBarLabel: "Search", headerTitle: "Search",
```

Comment:
We may want to make the search icon some sort of magnifying glass instead of a list like the recipe lists

Reply:
Kassi and I can look into it when we are working on the app UI this weekend if you would like

Reply:
I agree with alfred, i think that a magnifying glass makes more intuitive sense for the user